engine/index.js

```diff
@@ -462,3 +462,8 @@ export default {
   buildStatus,
   buildIndex,
 };
+
+const test = {
```

Comment:
You could try using `rewire` to avoid exporting internal functions.

https://github.com/jhnns/rewire